app/assets/stylesheets/rad_common/rad_bootstrap_overrides.scss

```diff
@@ -1,18 +1,27 @@
 html {
-  // Bootstrap 4 increased the default font size,
-  // so this just brings it back down so everything is closer to what it was.
+  /* Bootstrap 4 increased the default font size,
```

Comment:
Standards seem to dictate otherwise... no problem changing if you want to do your own thing.

https://developer.mozilla.org/en-US/docs/Web/CSS/Comments
https://www.w3schools.com/css/css_comments.asp

Reply:
@kevinh-csalabs we are not using hound for stylesheets nor JS, it's not configured the way we like it - just ignore for now